rust/headless-client/src/lib.rs

```diff
@@ -29,11 +29,10 @@ use platform::Signals;
 pub(crate) mod device_id;
 // Pub because the GUI reads the system resolvers
 pub mod dns_control;
-pub mod heartbeat;
 mod ipc_service;
 pub mod known_dirs;
 mod standalone;
-mod uptime;
+pub mod uptime;
```

Comment:
Should this move to `firezone-bin-utils` perhaps?

Reply:
Yeah. There's a few things in `headless-client` that gets re-used by the GUI client because they don't really fit into `connlib-shared` or `connlib-client-shared` (no other Client needs them) and I wasn't sure if they should break out into another crate.

Reply:
Unless you meant `firezone-cli-utils`, which is shared with the Gateway?

Reply:
> Unless you meant `firezone-cli-utils`, which is shared with the Gateway?

Yeah that one :)

Reply:
> and I wasn't sure if they should break out into another crate.

The purist in me wants to make them separate crates but that seems a bit excessive. I think we can get away with a "shared" crate that only top-level binaries have access to. So maybe we can rename `cli-utils` to `bin-shared` or something for anything that is shared between binaries?

That one shouldn't depend on connlib itself though to make sure it can be built early on in the dependency graph.